src/admin-api.ts

```diff
@@ -374,6 +374,16 @@ export class SPVWalletAdminAPI {
     return await this.http.adminRequest('/admin/contacts/confirmations', 'POST', { paymailA, paymailB });
   }
 
+  /**
+   * Unconfirm contact
+   *
+   * @param {string} id - ID of the contact to unconfirm
+   * @returns {Promise<void>}
+   */
+    async unconfirmContact(id: string): Promise<void> {
```

Comment:
Maybe modify a bit manage-contacts.ts example and add this to the flow somehow? Maybe in cleanup section? 🤔